web/server/entrypoint_test.go

```diff
@@ -445,8 +472,9 @@ func TestTunnelE2E(t *testing.T) {
 							Port: destPort, // allow tunneling to destination port
 						},
 						{
-							Port:              65535,           // allow tunneling to 65535
-							ConnectionTimeout: time.Nanosecond, // specify an impossibly small timeout
+							Port: timeoutDestPort, // allow tunneling to 65534
+							// specify a negative timeout since somehow 1 ns succeeds on windows sometimes
```

Comment:
not sure how 1ns succeeds 🤷 - this does mean if we add validation around this field the test may start failing